Getting-Started-with-Akka-http-signature.md

```diff
@@ -0,0 +1,250 @@
+
+***
+
+## Running the server
+
+In order to run a local server one must first locally clone the [Read-write-web play repository](https://github.com/read-write-web/rww-play). After that the user must navigate to their local Rww-play directory and run sbt there. For more information on how to run the server and the exact command used refer to the [Rww-play README page](https://github.com/read-write-web/rww-play).
+
+Users can also publish on their local Read-write-web play. In order to do this, a user must run the following commands inside the local rww-play directory.
+
+```bash
+$ sbt 
+> publishLocal
+```
+
+One can then import it into their scripts from inside the ammonite shell. The user should then be able to make a GET request on their local rww-play and check whether they can access a resource that has access control. This means that the user will need to make themselves a key.
+
+## The Key Pair
+
+The Akka-Http-Signature library uses asymmetrical cryptography. As detailed in the [Public-Key cryptography wikipedia page](https://en.wikipedia.org/wiki/Public-key_cryptography), this is a cryptographic system that uses pairs of private and public keys. Public keys can be looked up by anyone and private keys are only known by the owner. Communication is this system is achieved the following way: 
+1. The sender encrypts a message using the public key of the receiver
+1. The receiver gets the message and can decrypt it by using his own private key.
+The message sent can only be decrypted via the private key of the receiver.
```

Comment:
There are two ways to use symmetric key encryption. One of them to encrypt messages, which you describe here shortly. The other to sign messages. We are using the second one in HTTP-Signature to sign the headers of HTTP messages. So you chose the wrong one.